### ISSUE TITLE
Is there any plan for androidArm64?

### ISSUE THREAD

**Issue by RitwikSaikia** [Kotlin/kotlinx.coroutines#2083]
https://github.com/Kotlin/kotlinx.coroutines/blob/1.3.7/gradle/compile-native-multiplatform.gradle#L20

```kotlin
targets {
        if (project.ext.ideaActive) {
            fromPreset(project.ext.ideaPreset, 'native')
        } else {
            addTarget(presets.linuxX64)
            addTarget(presets.iosArm64)
            addTarget(presets.iosArm32)
            addTarget(presets.iosX64)
            addTarget(presets.macosX64)
            addTarget(presets.mingwX64)
            addTarget(presets.tvosArm64)
            addTarget(presets.tvosX64)
            addTarget(presets.watchosArm32)
            addTarget(presets.watchosArm64)
            addTarget(presets.watchosX86)
        }
    }
```

**Comment by lamba92** [comment]
Also, the `linuxArm64` and `linuxArm32` presets may be built but not published on Bintray :(
The [latest Gradle Metadata](https://kotlin.bintray.com/kotlinx/org/jetbrains/kotlinx/kotlinx-coroutines-core/1.3.7-1.4-M2/kotlinx-coroutines-core-1.3.7-1.4-M2.module) published for version `1.3.7-1.4-M2` don't even mention those platforms!

**Comment by elizarov** [comment]
This is a duplicate of #855. Please, follow up there.
